tests/test_reason_paths.py

```
@@ -0,0 +1,219 @@
+        fork_from=None,
+    )
+    p2 = ConsequencePath(
+        belief_ids=("A", "B"),
+        edge_kinds=(EDGE_RELATES_TO,),
+        compound_confidence=0.6,
+        weakest_link_belief_id="B",
+        fork_from=None,
+    )
+    assert p1 == p2
+    assert hash(p1) == hash(p2)
+    with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)